float32 flPathStartX
float32 flPathStartY
float32 flPathEndX
float32 flPathEndY
---
bool boSuccess